specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json

```diff
@@ -0,0 +1,2167 @@
+              "format": "date-time"
+            },
+            "hirStage": {
+              "type": "string",
+              "description": "Stage for HIR Document"
+            },
+            "additionalInformation": {
+              "type": "object",
+              "description": "Additional information",
+              "properties": {
+                "message": {
+                  "type": "string",
+                  "description": "Additional information Message"
+                }
+              }
+            },
+            "duration": {
+              "type": "integer",
+              "description": "duration in seconds",
+              "format": "int32"
+            },
+            "impactType": {
+              "type": "string",
+              "description": "The type of the impact"
+            },
+            "maintenanceId": {
+              "type": "string",
+              "description": "Unique Id for Planned maintenance event"
+            },
+            "maintenanceType": {
+              "type": "string",
+              "description": "The type of Planned maintenance event"
+            },
+            "argQuery": {
+              "type": "string",
+              "description": "ARG Query to fetch the affected resources from their existing ARG locations"
+            }
+          }
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ProxyResource"
+        }
+      ]
+    },
+    "link": {
+      "type": "object",
+      "description": "Useful links for service health event.",
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "Type of link.",
+          "enum": [
+            "Button",
+            "Hyperlink"
+          ],
+          "x-ms-enum": {
+            "name": "LinkTypeValues",
+            "modelAsString": true
+          }
+        },
+        "displayText": {
+          "type": "object",
+          "description": "Display text of link.",
+          "properties": {
+            "value": {
+              "type": "string",
+              "description": "Display text of link."
+            },
+            "localizedValue": {
+              "type": "string",
+              "description": "Localized display text of link."
+            }
+          }
+        },
+        "extensionName": {
+          "type": "string",
+          "description": "It provides the name of portal extension to produce link for given service health event."
+        },
+        "bladeName": {
+          "type": "string",
+          "description": "It provides the name of portal extension blade to produce link for given service health event."
+        },
+        "parameters": {
+          "type": "object",
+          "description": "It provides a map of parameter name and value for portal extension blade to produce lik for given service health event."
```

Comment:
typo 'lik' - I wonder if its not caught by spellcheck. Should it be link?

Reply:
I will fix this,  but will check if this will cause any breaking changes if I change this. And then proceed accordingly.